src/components/icon/icon.jsx

```diff
@@ -23,13 +23,13 @@ Icon.propTypes = {
 };
 
 Icon.defaultProps = {
-  fill: '#222',
-  stroke: '#222',
+  fill: 'currentColor',
+  stroke: 'currentColor',
   'aria-hidden': 'true',
   focusable: 'false',
   strokeWidth: 1,
-  height: 16,
-  width: 16,
+  height: '1em',
```

Comment:
Unfortunately this will break the `Ellipse` icon. However I think that it is better to default it to `1em` like you did here so to fix the error we can add `height: 16, width: 16` to `Ellipse` defaultProps instead!

Reply:
Fixed on Ellipse and no console errors when viewing it locally so this should be ok now.